sdk/loadtestservice/azure-mgmt-loadtesting/tests/test_resourceoperations.py

```diff
@@ -0,0 +1,161 @@
+        
+        # Get the result of the poller
+        loadtest_resource = loadtest_resource_poller.result()
+
+        # Assert the result of the poller
+        assert loadtest_resource != None
+        assert loadtest_resource.id != None
+        assert loadtest_resource.name == resource_group.name+"resource"
+        assert loadtest_resource.location == self.AZURE_LOCATION
+        assert loadtest_resource.identity != None
+        assert loadtest_resource.identity.type == "SystemAssigned, UserAssigned"
+        assert len(loadtest_resource.identity.user_assigned_identities) == 2
+        assert loadtest_resource.encryption != None
+        assert loadtest_resource.encryption.key_url == self.key.id
+        assert loadtest_resource.encryption.identity != None
+        assert loadtest_resource.encryption.identity.type == "UserAssigned"
+        assert loadtest_resource.encryption.identity.resource_id == self.msi1.id
+        
+        # # Get the load test resource
+        loadtest_resource_get = self.loadtestservice_client.load_tests.get(resource_group.name, resource_group.name+"resource")
+        
+        # # Assert the result of the get operation
+        assert loadtest_resource_get != None
+        assert loadtest_resource_get.id != None
+        assert loadtest_resource_get.name == resource_group.name+"resource"
+        assert loadtest_resource_get.location == self.AZURE_LOCATION
+        assert loadtest_resource_get.identity != None
+        assert loadtest_resource_get.identity.type == "SystemAssigned, UserAssigned"
+        assert len(loadtest_resource_get.identity.user_assigned_identities) == 2
+        assert loadtest_resource_get.encryption != None
+        assert loadtest_resource_get.encryption.key_url == self.key.id
+        assert loadtest_resource_get.encryption.identity != None
+        assert loadtest_resource_get.encryption.identity.type == "UserAssigned"
+        assert loadtest_resource_get.encryption.identity.resource_id == self.msi1.id
+
+
+        # Update the load test resource
+        loadtestresourcePatchdata = LoadTestResourcePatchRequestBody(
+            identity=ManagedServiceIdentity(
+                type="SystemAssigned,UserAssigned",
+                user_assigned_identities={
+                    self.msi2.id: None
+                }
+            ),
+        )
+
+        # load test resource update begin - returns a poller
+        loadtest_resource_patch_poller = self.loadtestservice_client.load_tests.begin_update(resource_group.name, resource_group.name+"resource", loadtestresourcePatchdata)
+        
+        # Wait for the poller to complete
+        while loadtest_resource_poller.done()==False:
+            loadtest_resource_poller.wait(2)
+        
+        # Get the result of the poller
+        loadtest_resource_patch_response = loadtest_resource_patch_poller.result()
+
+        # Assert the result of the poller
+        assert loadtest_resource_patch_response != None
```

Comment:
```suggestion
        assert loadtest_resource_patch_response
```
same for other lines

Reply:
Updated it at all required lines